src/App.js

```diff
@@ -1,25 +1,36 @@
-import logo from "./logo.svg";
-import "./App.css";
-import { useEffect, useState } from "react";
+import './App.css';
+import { useEffect, useState } from 'react';
 
 function App() {
-  const [data, setData] = useState(null)
+  const [data, setData] = useState({});
   useEffect(() => {
     async function loadData() {
-      const response = await fetch('https://redi-final-restaurants.herokuapp.com/restaurants')
-      const data = await response.json()
-      setData(data)
+      const response = await fetch(
+        'https://redi-final-restaurants.herokuapp.com/restaurants'
+      );
+      const data = await response.json();
+      setData(data);
     }
-    loadData()
-  }, [])
-  console.log(data)
+    loadData();
+  }, []);
+  console.log(data);
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>Hello Team 1!</h1>
-      </header>
+    <div className='App'>
+      {data.results.map((rest) => (
```

Comment:
'rest' is not a descriptive name, because it is easily confused with things like `const [first, second, ...rest] = props`
either use `restaurant` or it is common to use `item` for each item in an array